pkg/kubeapiserver/authorizer/config.go

```diff
@@ -105,11 +111,32 @@ func (config AuthorizationConfig) New() (authorizer.Authorizer, authorizer.RuleR
 			if config.WebhookConfigFile == "" {
 				return nil, nil, errors.New("Webhook's configuration file not passed")
 			}
-			webhookAuthorizer, err := webhook.New(config.WebhookConfigFile,
+			// Load config
+			loadingRules := clientcmd.NewDefaultClientConfigLoadingRules()
+			loadingRules.ExplicitPath = config.WebhookConfigFile
+			loader := clientcmd.NewNonInteractiveDeferredLoadingClientConfig(
+				loadingRules, &clientcmd.ConfigOverrides{})
+			clientConfig, err := loader.ClientConfig()
+			if err != nil {
+				return nil, nil, fmt.Errorf("invalid webhook config: %v", err)
+			}
+
+			// Set timeout and dialer
+			clientConfig.Timeout = 30 * time.Second
+			clientConfig.Dial = config.WebhookDialer
+
+			// Build client
+			webhookClient, err := authorizationclient.NewForConfig(clientConfig)
+			if err != nil {
+				return nil, nil, fmt.Errorf("while creating webhook client: %v", err)
+			}
+
+			webhookAuthorizer, err := webhook.NewFromInterface(
+				webhookClient.SubjectAccessReviews(),
```

Comment:
This client automatically sets the URL path. See the existing webhook authorizer for how to create a client that sends access reviews to the root path https://github.com/kubernetes/kubernetes/blob/9b8d70dc426f7203f2a32e4ec0e4117c204fee0d/staging/src/k8s.io/apiserver/plugin/pkg/authorizer/webhook/webhook.go#L235-L245